.github/workflows/ispc-ci.yml

```diff
@@ -53,7 +55,9 @@ jobs:
     runs-on: ubuntu-22.04
     outputs:
       tests_matrix_targets: ${{ steps.set-flow.outputs.matrix }}
```

Comment:
Should we rename add `_x86` suffix here?

Reply:
Done